README.md

```diff
@@ -6,6 +6,35 @@ The volcano dashboard provides a basic dashboard that can be easily deployed in
 
 <img src="docs/images/demo.gif" alt="volcano dashboard" style="zoom:50%;" />
 
+## Getting Started
+
+Make sure [`node.js`](https://nodejs.org/en/download) is installed on your system and we prefer [`visual-studio-code`](https://code.visualstudio.com/download) as IDE.
```

Comment:
This phrase should be put at `### Development` in CONTRIBUTING.md.

Reply:
Sorry for not make it clear, this section should be remain in `CONTRIBUTING.md` at  `### Development` part.